rfd/0037-desktop-access-protocol.md

```diff
@@ -200,11 +210,28 @@ This message contains a mouse wheel update. Sent from client to server.
 #### 9 - error
 
 ```
-| message type (9) | message_length uint32 | message []byte
+| message type (9) | message_length uint32 | message []byte |
 ```
 
 This message indicates an error has occurred.
 
+#### 28 - notification
+
+```
+| message type (28) | message_length uint32 | message []byte | severity byte
```

Comment:
message_length includes the severity byte, right? or is it the length of the string?

Let's specify since it's unclear what you mean by "message" here - it could be the "notification message" or it could be the "message field" in the "notification message"

Reply:
It's just the length of the string. We're not entirely consistent within this RFD, but in 0067 I began always calling the length of `<name> []byte` fields `<name>_length`. E.g. see

```
| message type (21) | completion_id uint32 | directory_id uint32 | path_length uint32 | path []byte | offset uint64 | write_data_length uint32 | write_data []byte |
```

I'll add a note here to clarify.